silkworm/db/snapshots/repository.hpp

```diff
@@ -73,13 +74,39 @@ class SnapshotRepository {
     [[nodiscard]] std::vector<std::shared_ptr<IndexBuilder>> missing_indexes() const;
     void remove_stale_indexes() const;
 
-    auto view_bundles() const { return make_map_values_view(bundles_); }
-    auto view_bundles_reverse() const { return std::ranges::reverse_view(view_bundles()); }
+    using TBundles = std::map<BlockNum, std::shared_ptr<SnapshotBundle>>;
```

Comment:
We should rename `TBundles` as `Bundles` here because it's not a template typename like e.g. `TBaseView`. I know it's a minor thing, but I casually started reading from the cpp file and couldn't find what I expected to be a template parameter.